src/packet/bwe/arrival_group.rs

```diff
@@ -1,11 +1,14 @@
+use std::cmp::Ordering;
 use std::mem;
 use std::time::{Duration, Instant};
 
 use crate::rtp_::SeqNo;
 
 use super::AckedPacket;
 
-const BURST_TIME_INTERVAL: Duration = Duration::from_millis(5);
+const BURST_TIME_INTERVAL: f64 = 0.005; // 5ms
```

Comment:
I think we should use Duration, it's a richer type which is less prone to mistakes

Reply:
`Duration::as_secs_f64` is not `const` on current MSRV :(

[CI run](https://github.com/algesten/str0m/actions/runs/12735618126/job/35494573846):
```
error: `Duration::as_secs_f64` is not yet stable as a const fn
Error:  --> src/packet/bwe/arrival_group.rs:9:34
  |
9 | const BURST_TIME_INTERVAL: f64 = Duration::from_millis(5).as_secs_f64();
  |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: could not compile `str0m` (lib) due to previous error
```

Reply:
Ah! Leave a comment for what MSRV we can switch.

Reply:
I'm not suggesting `Duration::from_millis(5).as_secs_f64()`. We don't need Duration to transform `5` into `0.005`. My point is to try and use Durations rather than converting to seconds at all, if we must convert to seconds we can use `as_secs_f64` at the callsite

Reply:
Ah, right, it just seemed like a minor inefficency, but whatever, fixed!